src/wp-admin/options-reading.php

```diff
@@ -80,10 +87,40 @@
 	</label>
 	</p>
 <ul>
-	<li><label for="page_on_front"><?php printf( __( 'Homepage: %s' ), wp_dropdown_pages( array( 'name' => 'page_on_front', 'echo' => 0, 'show_option_none' => __( '&mdash; Select &mdash;' ), 'option_none_value' => '0', 'selected' => get_option( 'page_on_front' ) ) ) ); ?></label></li>
-	<li><label for="page_for_posts"><?php printf( __( 'Posts page: %s' ), wp_dropdown_pages( array( 'name' => 'page_for_posts', 'echo' => 0, 'show_option_none' => __( '&mdash; Select &mdash;' ), 'option_none_value' => '0', 'selected' => get_option( 'page_for_posts' ) ) ) ); ?></label></li>
+	<li><label for="page_on_front">
+	<?php
+	printf(
+		__( 'Homepage: %s' ),
+		wp_dropdown_pages(
+			array(
+				'name'              => 'page_on_front',
+				'echo'              => 0,
+				'show_option_none'  => __( '&mdash; Select &mdash;' ),
+				'option_none_value' => '0',
+				'selected'          => get_option( 'page_on_front' ),
+			)
+		)
+	);
+	?>
+									</label></li>
+	<li><label for="page_for_posts">
+	<?php
+	printf(
+		__( 'Posts page: %s' ),
+		wp_dropdown_pages(
+			array(
+				'name'              => 'page_for_posts',
+				'echo'              => 0,
+				'show_option_none'  => __( '&mdash; Select &mdash;' ),
+				'option_none_value' => '0',
+				'selected'          => get_option( 'page_for_posts' ),
+			)
+		)
+	);
+	?>
+									</label></li>
```

Comment:
indentation

Reply:
Agreed, will add to the amends already made locally ready to update the PR.